apps/sunrisesunset/sunrise_sunset.star

```diff
@@ -126,6 +175,20 @@ def get_schema():
                 desc = "Location for which to display the sun rise and set times.",
                 icon = "place",
             ),
+            schema.Toggle(
```

Comment:
One thought I have: If we made this a dropdown, we could eliminate the case where both sunrise and sunset would be toggled off. It could default to showing both and have options to select either or.

Reply:
🤦 I missed the Dropdown item in Schema. I'll do another pass 👍

Reply:
OK, should be resolved with 8f6972beea61184dd90422d9f1eec6aa171069db